client/types/actionStrings.ts

```diff
@@ -0,0 +1,2 @@
+export const addPerson = "SET_NAMES"
```

Comment:
why are we separating these? don’t think they’ll take too much space up for now. makes it harder to read